### ISSUE TITLE
epic: Solid.js client

### ISSUE THREAD

**Issue by insertish** [revoltchat/revoltchat-solid-client-monorepo#14]
Milestones:
- [Authentication (20%)](https://github.com/revoltchat/revoltchat-solid-client-monorepo/milestone/1)
- [Interface (10%)](https://github.com/revoltchat/revoltchat-solid-client-monorepo/milestone/2)
- [Messaging (10%)](https://github.com/revoltchat/revoltchat-solid-client-monorepo/milestone/3)
- Settings (0%)
- Server / Channel Settings (0%)

**Comment by insertish** [comment]
Added a link to point to the wiki for a feature matrix.

**Comment by AmyNicole8715** [comment]
I am not seeing anything with help wanted on it

**Comment by insertish** [comment]
Added a couple issues to help wanted, will be creating more now too for those which are missing from the feature matrix.

**Comment by nlutterman** [comment]
Just to double check before I start digging through stuff and get all annoying and in the way, are you only interested in receiving contributions from people outside of your organization for issues tagged `help-wanted`?

Or is it reasonable to attempt tackling an issue that isn't tagged, but is an area one has experience with?

**Comment by Rexogamer** [comment]
You should be fine to go for any unclaimed issues at this point

**Comment by tilkinsc** [comment]
Me and my colleague are trying to get started with revolt, but were having strange bugs with it (like audio connection). We are using the docker container. We are okay with the website revolt. What is your recommended course of action to use it as nightly?

**Comment by Rexogamer** [comment]
@tilkinsc: it seems like you want revoltchat/self-hosted instead, although heads up that we're reworking voice support at the moment

**Comment by Arkanu-of-Galatiel** [comment]
Will this still be compatible with windows 7? That's one thing I appreciate about Revolt as it stands now.

**Comment by Rexogamer** [comment]
as far as I can tell, Tauri supports any version of Windows that can run Edge WebView2 - the last version that supports Windows 7 is v109 (per [Microsoft's documentation](https://learn.microsoft.com/en-us/microsoft-edge/webview2/#windows-7-and-8)), so I *think* you'll be able to run it as long as you have WebView2 installed?

(I will say, though, that I wouldn't advise using Windows 7 at this point - 10 should support anything that runs 7, and if you don't trust 10/11 then there are a variety of Linux distributions that should run well on your device)

**Comment by wuemeli** [comment]
hypee

**Comment by sovos15** [comment]
Things still moving?

**Comment by theAkito** [comment]
Is this still alive?

**Comment by insertish** [comment]
yes im trying to onboard more maintainers too to help things along
i am very very busy with uni, other life things, & other revolt things rn

**Comment by UMCEKO** [comment]
Do you guys have a discord? I am willing to help with the development.

**Comment by sharkaccino** [comment]
there is a discord, but its largely abandoned as far as i can tell. the vast majority of development happens here on github or on revolt itself

**Comment by ShugarSkull** [comment]
I also would like to help, is there an alpha version working with tauri already ? I did not find it

**Comment by insertish** [comment]
It lives in `packages/desktop`. I think it needs updating.

**Comment by halian** [comment]
@insertish there is no `packages/desktop` directory in the repository, only `package/revolt-desktop-git`, and the website throws a 404 when I try to access the latter.

**Comment by insertish** [comment]
> [@insertish](https://github.com/insertish?rgh-link-date=2025-03-22T02%3A40%3A04.000Z) there is no `packages/desktop` directory in the repository, only `package/revolt-desktop-git`, and the website throws a 404 when I try to access the latter.

https://github.com/revoltchat/frontend/tree/main/packages/desktop

**Comment by ghost** [comment]
is this still ongoing or what?

**Comment by insertish** [comment]
> is this still ongoing or what?

yes

**Comment by cdobrich** [comment]
Sorry, but I'm confused. Where is the best place to find, track, and create features? (Stoat Development Chat room, GitHub issues+sub-issues?) What's the criteria for creating a feature request?

For example, if I wanted to make an issue for Attachment features, where should that be? Specifically I would like to request (or someone personally attempt to write) a feature that, for a single post, groups multiple attachments together. It also provides a carousel sort of viewer, with left-arrow + right-arrow and support for key input. (Basically what Discord currently does.)

![Image](https://github.com/user-attachments/assets/1ccfed88-376b-4780-b485-f89454249f92)

![Image](https://github.com/user-attachments/assets/aefe0081-d0dc-4b2c-8c6b-f9e855545acf)

**Comment by ppabst** [comment]
Is there any progress on this project? Or is it unfortunately dead?

**Comment by insertish** [comment]
> Is there any progress on this project?

Yes.

> Or is it unfortunately dead?

No.

**Comment by homersimpsons** [comment]
Correct link for the matrix (comparison with the old client): https://stoatchat.github.io/for-web/feature-matrix.html

Note that https://wiki.rvlt.gg/index.php/Feature_Matrix mentions that the documentation was moved to https://revoltchat.github.io/frontend/feature-matrix.html but this was probably the repository before the rename (this link can also be updated)

**Comment by xksteven** [comment]
Out of curiosity is there a stance or policy for submitting code generated by or in assistance with AI?

**Comment by Zomatree** [comment]
We do not allow any AI generated code or contributions.
